website/docs/docs/building-a-dbt-project/package-management.md

```diff
@@ -122,6 +122,10 @@ packages:
   - git: "https://{{env_var('GIT_CREDENTIALS')}}@github.com/fishtown-analytics/dbt-utils.git" # git HTTPS URL
 ```
 
+**Please note**:
+  - We recommend using the [git URL](http://localhost:3000/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-import-a-project-by-git-url#github) clone strategy in order to use private packages. If you'd like to use the Github app clone strategy, please contact support.
```

Comment:
The git url approach would not be using the repository as a package but rather as a project so this note doesn't entirely make sense.